### DISCUSSION TITLE
error  salt-master

### DISCUSSION THREAD

**Discussion by wangyan-samuel** [Security-Onion-Solutions/securityonion#13326] [2.4]
### Version

2.4.80

### Installation Method

Network installation on Ubuntu

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

16

### RAM

32G

### Storage for /

500G

### Storage for /nsm

400G+

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

<img width="584" alt="0ff6d9f4e7452db1e8d68797e38aeb1" src="https://github.com/Security-Onion-Solutions/securityonion/assets/85142515/396f8d28-a39e-473a-8a4f-f12a58f3f98d">

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
Is the salt-master service running? `sudo systemctl status salt-master`
